internal/cephfs/groupcontrollerserver_test.go

```diff
@@ -106,13 +106,12 @@ func TestControllerServer_validateCreateVolumeGroupSnapshotRequest(t *testing.T)
 		},
 	}
 	for _, tt := range tests {
-		ts := tt
-		t.Run(ts.name, func(t *testing.T) {
+		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
```

Comment:
Same as above comment.
We'll need the local in loop variable since the test will be run in parallel.